user/models.py

```diff
@@ -1,5 +1,52 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import (
+    AbstractUser,
+    UserManager as DjangoUserManager
+)
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class UserManager(DjangoUserManager):
+    """Define a model manager for User model with no username field."""
+
```

Comment:
I don't even know if there is any point in deleting this, but I'll make a note)
I mean docstrings

Reply:
I believe it's acceptable to leave this comment as we include additional information about why we implemented this class